browser/src/control/jsdialog/Widget.ColorPicker.ts

```diff
@@ -103,9 +111,19 @@ function createColor(
 	color.setAttribute('name', colorItem);
 	color.setAttribute('index', index);
 	color.tabIndex = 0;
+	color.innerHTML = isCurrent ? '&#149;' : '&#160;';
 	if (themeData) color.setAttribute('theme', themeData);
 
-	color.innerHTML = isCurrent ? '&#149;' : '&#160;';
+	// Set color tooltips
+	if (themeData) {
+		const themeColors: ThemeColor[][] =
+			window.app.colorPalettes['ThemeColors'].colors;
+		const flattened: ThemeColor[] = themeColors.flat();
+		const found = flattened.find((item) => item.Value === colorItem);
+		if (found) color.setAttribute('data-cooltip', found.Name);
+	} else if (window.app.colorNames)
+		color.setAttribute('data-cooltip', window.app.colorNames[colorItem]);
```

Comment:
can we have `window.app.colorNames[colorItem].Name` for unification ?
in case we will need more data for each color in the future (porosed on core side)